packages/requisitions/src/ResponseRequisition/DetailView/SidePanel.tsx

```diff
@@ -49,11 +54,64 @@ const AdditionalInfoSection: FC = () => {
   );
 };
 
+const RelatedDocumentsRow: FC<{
+  label: string;
+  to: string;
+  value?: number | null;
+}> = ({ label, to, value }) => (
+  <PanelRow>
+    <PanelLabel>{label}</PanelLabel>
+    <PanelField>
+      <Link to={to}>{`#${value}`}</Link>
+    </PanelField>
+  </PanelRow>
+);
+
 const RelatedDocumentsSection: FC = () => {
   const t = useTranslation('distribution');
+  const d = useFormatDate();
+  const { shipments } = useResponseFields('shipments');
+
+  const getTooltip = (createdDatetime: string, username?: string) => {
+    let tooltip = t('messages.outbound-shipment-created-on', {
+      date: d(new Date(createdDatetime)),
+    });
+
+    if (username && username !== 'unknown') {
+      tooltip += t('messages.by-user', { username });
```

Comment:
```suggestion
      tooltip += ` ${t('messages.by-user', { username })}`;
```

just the same as request, needs a space in here